.github/workflows/close-issues-on-pr-merge.yml

```
@@ -0,0 +1,113 @@
+            }
+
+            // Process each issue
+            for (const issueNumber of allIssues) {
+              try {
+                // Check if issue exists and is open
+                const { data: issue } = await github.rest.issues.get({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: issueNumber
+                });
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)